File: HammingDistance.py
class Solution:
    def hammingDistance(self, x: int, y: int) -> int:
        count = 0
        n = x ^ y
        while n:
            count += n & 1
            n >>= 1
        return count


if __name__ == '__main__':
    x = int(input())
    y = int(input())
    solution = Solution()
    print(solution.hammingDistance(x, y))

File: HammingDistance.cpp
#include<iostream>

using namespace std;

class Solution {
public:
    int hammingDistance(int x, int y) {
        int count = 0;
        while(x || y) {
        	count += (x ^ y) & 1;
        	x >>= 1;
        	y >>= 1;
		}
		return count;
    }
};

int main() {
	int x, y;
	cin>>x>>y;
	
	Solution *solution = new Solution();
	cout<<solution->hammingDistance(x, y);
	
	return 0;
} 
